lib/trie/lookup.go

```diff
@@ -49,5 +49,5 @@ func find(parent Node, key []byte, recorder recorder) error {
 		return nil
 	}
 
-	return find(b.Children[key[length]], key[length+1:], recorder)
+	return find(b.Children[key[length]], key[length+1:], recorder, false)
```

Comment:
```suggestion
	const isChildRoot = false
	return find(b.Children[key[length]], key[length+1:], recorder, isChildRoot)
```

Reply:
the `isChildRoot` variable does not make much sense, the root node already has been processed and the other nodes are not the root, so it is explicit that for other `find` calls the `false` argument means the nodes are not the root